apps/dsp-app/src/app/project/ontology/property-form/property-form.component.ts

```diff
@@ -828,6 +829,7 @@ export class PropertyFormComponent implements OnInit {
                     this.lastModificationDate = res.lastModificationDate;
                     // close the dialog box
                     this.loading = false;
+                    this.submittingChange = false;
                     this.closeDialog.emit();
                 },
                 (error: ApiResponseError) => {
```

Comment:
I think you should also set the submittingChange to false when you get in to the error body

Reply:
Good point as it is not submitting anymore. But if there is an error we don't want a user to retry the same request which resulted in an error (retries should be introduced automatically).

So I prefer to close the whole dialog and destroy the whole property component. I'll change that.